app/code/Tasks/FeedbackManager/Ui/Component/ApproveButton.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Tasks\FeedbackManager\Ui\Component;
+
+use Magento\Framework\View\Element\UiComponent\Control\ButtonProviderInterface;
+
+class ApproveButton extends GenericButton implements ButtonProviderInterface
+{
+
+    /**
+     * Retrieve button-specified settings
+     *
+     * @return array
+     */
+    public function getButtonData()
+    {
+        $data = [];
+        $status = $this->getStatus();
+        if(!$status || ($status ==2)) // Showing 'Approve' button for pending and declined statuses
```

Comment:
@gayathri-aravind Always use strict comparison

Reply:
Strict comparison used to check the status. Commit: 3a2a64414336d13c9a2efa6d09126bb212d438a9